pkg/etcd-backup-operator/controller/s3backup.go

```diff
@@ -0,0 +1,232 @@
+package controller
+
+import (
+	"context"
+	"crypto/tls"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"os"
+	"path"
+	"path/filepath"
+
+	"github.com/Sirupsen/logrus"
```

Comment:
`github.com/coreos/etcd-operator` imports should be grouped together and be separate from non etcd-operator related imports.

Reply:
good catch. easy to miss those

Reply:
done!